backend-api/src/functions/asyncBiometricToken/registeredLogs.ts

```diff
@@ -4,8 +4,11 @@ import {
 } from "../services/logging/commonRegisteredLogs";
 import { RegisteredLogMessages } from "../services/logging/types";
 
-export type MessageName = CommonMessageNames;
+export type MessageName = "REQUEST_BODY_INVALID" | CommonMessageNames;
 
 export const registeredLogs: RegisteredLogMessages<MessageName> = {
+  REQUEST_BODY_INVALID: {
+    messageCode: "MOBILE_ASYNC_REQUEST_BODY_INVALID",
```

Comment:
Would `MOBILE_ASYNC_BIOMETRIC_TOKEN_REQUEST_BODY_INVALID` be better here? We'll have different endpoints validating different request bodies.

Reply:
Done